lms/djangoapps/certificates/views/webview.py

```diff
@@ -476,6 +477,7 @@ def render_cert_by_uuid(request, certificate_uuid):
     template_path="certificates/server-error.html",
     test_func=lambda request: request.GET.get('preview', None)
 )
+@login_required
```

Comment:
Why do we protect this public view while don't protect render_cert_by_uuid (line 462)?